src/Files.App/Actions/Content/Run/RunWithPowershellAction.cs

```diff
@@ -0,0 +1,47 @@
+﻿using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.DependencyInjection;
+using Files.App.Commands;
+using Files.App.Contexts;
+using Files.App.Extensions;
+using Files.App.Helpers;
+using Files.App.Shell;
+using Files.Backend.Helpers;
+using System.Threading.Tasks;
+
+namespace Files.App.Actions
+{
+	internal class RunWithPowershellAction : ObservableObject, IAction
+	{
+		public IContentPageContext context = Ioc.Default.GetRequiredService<IContentPageContext>();
+
+		public bool IsExecutable => context.SelectedItem is not null &&
+			FileExtensionHelpers.IsPowerShellFile(context.SelectedItem.FileExtension);
+
+		public string Label => "RunWithPowerShell".GetLocalizedResource();
+
+		public string Description => "TODO: Need to be described.";
+
```

Comment:
If you use `{ get; }` instead of `=>` we don't need to use `Mode=OneWay` in bindings (maybe `OneTime` could improve a little resource usage. Still, I'm not sure how that's implemented). Since this a fixed string I think we should go that way

Reply:
@0x5bfa can you change `=>` with `{ get; }`? (With `=>` you have to use `Mode=OneWay`)

Reply:
You mean `public string Label { get => "RunWithPowerShell".GetLocalizedResource(); }` or `public string Label { get } = "RunWithPowerShell".GetLocalizedResource();`?

Reply:
The second one

Reply:
I think not here, there're a lot to update to use this.

Reply:
What's the problem? `RunWithPowerShell` is not a formatted string

Reply:
I thought there's a template for Action class. Any class else has lambda expression. I wondered it is ok if changed only in this class.

Reply:
Still, if you use the lambda you have to add back `Mode=OneWay` to the bindings, if not they won't work

Reply:
Anyway, I think we should change the lambda in most of the actions (except the ones where the label might change [Archives, ...])

Reply:
Thank you, I'll change them:)